File: src/pipCoding/code_java/Ch1_isNumberIsPrime.java
package pipCoding.code_java;

import java.util.Scanner;

public class Ch1_isNumberIsPrime {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n1 = scn.nextInt();

        for (int i = 0; i < n1; i++) {
            int n = scn.nextInt();

            int count = 0;
            for (int div = 1; div <= n; div++) {
                if (n % div == 0) {
                    count++;
                }
            }

            if (count == 2) {
                System.out.println("No is Prime");
            } else {
                System.out.println("No is Not Prime");
            }
        }
    }
}

File: src/pipCoding/code_kotlin/ch1_isNumberIsPrime.kt
package pipCoding.code_kotlin

import java.util.*
import kotlin.math.sqrt

fun main() {
    val scn = Scanner(System.`in`)
    val n = scn.nextDouble()
    isNumberIsPrime(n)
}

fun isNumberIsPrime(n: Double) {
    if (n == 0.0 || n == 1.0) {
        println("Number is not Prime")
    }

    if (n > 1) {
        for (i in 2 until sqrt(n).toInt()) {
            if (n % i == 0.0) {
                println("number is not Prime")
                return
            }
        }
        println("number is Prime")
    }
}
